### PR TITLE
Add more projects to docs/projects

### PR DESCRIPTION
**Pull request by kymppi**
Added projects:
- Koululaskuri
- Abitti OpenAccess

related to: #47

### CODE REVIEW

File: docs/projects/Koululaskuri/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Koululaskuri
+
+## Authors
+- Roni
+
+## Short description
+Koululaskuri on simppeli discord webhook joka kertoo kauanko koulua on jäljellä.
```
Comment (ahnl) [inline docs/projects/Koululaskuri/README.md:7]: Pay attention to the style of text

Comment (ahnl) [inline docs/projects/Koululaskuri/README.md:7]: Please pay attention to the style of text.

**Review by ahnl — CHANGES_REQUESTED** [review]

**Comment by Esinko** [comment]
@raikasdev Is this text correct? Please review it.

**Comment by raikasdev** [comment]
It is correct, but would like it to be more "formal" for the association homepage.

**Pull request closed without merging**